feelslikemonday/app/src/main/java/com/example/feelslikemonday/DAO/DAO.java

```diff
@@ -0,0 +1,94 @@
+package com.example.feelslikemonday.DAO;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.example.feelslikemonday.model.User;
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static androidx.constraintlayout.widget.Constraints.TAG;
+
+public class DAO<T,U> {
```

Comment:
Were you using this for testing purposes? Doesn't look like it belongs in the PR.

Reply:
I was hoping to save this away since I want to find a way to generalize the DAOs if time permits. Probably should just stash this, hey?